src/main/java/com/google/gcs/sdrs/dao/impl/RetentionRuleDaoImpl.java

```diff
@@ -206,6 +206,32 @@ public RetentionRule findGlobalRuleByProjectId(String projectId) {
     return result;
   }
 
+  @Override
+  public List<RetentionRule> findRulesByStorageRoot(
```

Comment:
what is a storageRoot - the bucket or a particular "path" prefix?  Either way can we add that in the method Javadoc?

Reply:
done